docs/api/portable-stories-playwright.md

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Portable stories in Playwright'
+---
+
+export const SUPPORTED_RENDERERS = ['react', 'vue'];
+
+(⚠️ **Experimental**)
+
+<If notRenderer={SUPPORTED_RENDERERS}>
+
+<Callout variant="info">
+
+Portable stories are currently only supported in [React](?renderer=react) and [Vue](?renderer=vue) projects.
+
+</Callout>
+
+<!-- End non-supported renderers -->
+
+</If>
+
+<If renderer={SUPPORTED_RENDERERS}>
+
+Portable stories are Storybook [stories](../writing-stories/index.md) which can be used in external environments, such as [Playwright Component Tests (CT)](https://playwright.dev/docs/test-components).
+
+Normally, Storybok composes a story and its [annotations](#annotations) automatically, as part of the [story pipeline](#story-pipeline). When using stories in Playwright CT, you can use the [`createTest`](#createtest) function, which extends Playwright's test functionality to add a custom `mount` mechanism, to take care of the story pipeline for you.
+
+<If renderer="react">
+
+<Callout variant="warning">
+
+**Using `Next.js`?** Next.js requires specific configuration that is only available in [Jest](./portable-stories-jest.md). The portable stories API is not supported in Next.js with Playwright CT.
+
+</Callout>
+
+</If>
+
+## createTest
+
+(⚠️ **Experimental**)
+
+Instead of using Playwright's own `test` function, you can use Storybook's special `createTest` function which will extend Playwright's test functionality to add a custom `mount` mechanism which will load, render, and play the story. This function is experimental and is subject to changes.
```

Comment:
```suggestion
Instead of using Playwright's own `test` function, you can use Storybook's special `createTest` function to extend Playwright's base fixture and override the `mount` function so it loads, renders, and plays the story. This function is experimental and is subject to changes.

Read more on extending base fixtures in [the Playwright documentation](https://playwright.dev/docs/test-fixtures#creating-a-fixture).
```

Reply:
Thanks! I tweaked the wording slightly, so didn't accept the suggestion directly.